lib/ex_l7/message/header.ex

```diff
@@ -0,0 +1,60 @@
+defmodule ExL7.Message.Header do
+  @moduledoc """
+  Helper functions for retrieving HL7 message header information
+  """
+
+  import ExL7.Query
+
+  def get_control_characters(%ExL7.Message{control_characters: chars}) do
+    get_control_characters(chars)
+  end
+
+  def get_control_characters(%ExL7.ControlCharacters{} = chars) do
+    chars.component <> chars.repeat <> chars.escape <> chars.sub_component
+  end
+
+  def get_sending_application(%ExL7.Message{} = l7_message) do
+    query(l7_message, "MSH|2")
+  end
+
+  def get_sending_facility(%ExL7.Message{} = l7_message) do
+    query(l7_message, "MSH|3")
+  end
+
+  def get_receiving_application(%ExL7.Message{} = l7_message) do
+    query(l7_message, "MSH|4")
+  end
+
+  def get_receiving_facility(%ExL7.Message{} = l7_message) do
+    query(l7_message, "MSH|5")
+  end
+
+  def get_date_time(%ExL7.Message{} = l7_message) do
+    # TODO: Date Time formatting
```

Comment:
I'm removing the TODO comment since I don't think it's necessary to format here. I believe the benefit to _not_ format is that whoever is using this can just call ExL7.Date.format themselves, and specify a specific format. I rather keep this call very simple and only return the tiny data element it refers to (for example, they might purposely want the default-formatted string).